_src/contact.jade

```diff
@@ -4,14 +4,33 @@
 .container
   .row
     .col-md-8
-      div
-        h2.subheader Get in touch
-        p codeforokc [at] gmail.com
-        p
-          a(href='https://twitter.com/codeforokc') @codeforokc
-          |  |
-          a(href='https://www.facebook.com/search/top/?q=code%20for%20okc') #codeforokc
-        p
-          a(href='https://www.facebook.com/CodeForOKC/')
-            | facebook.com/CodeForOKC
+        div
+          h2.subheader How would you use Open Budget?
```

Comment:
@mathewmorris can you adjust this to match the title of the site used everywhere else? "Open Budget: OKC" instead of just "Open Budget"